### ISSUE TITLE
Building for 32 bit win2k/xp

### ISSUE THREAD

**Issue by pachuco** [leecher1337/es1969#2]
This is compilable for 32 bit win2k/xp, yes?
Might be interesting to try in place of original.

**Comment by leecher1337** [comment]
Yes, can be compiled for 32bit too. But doesn't have any advantage over the original driver besides the ability to place your own bank file.

**Comment by pachuco** [comment]
I suppose advantage comes from having source to fiddle with.
Though, to take advantage, should have second PC with debug serial and soundcard.
Suffer not the bsod.

**Comment by Nable80** [comment]
> Though, to take advantage, should have second PC with debug serial and soundcard.

You can try to use SoftICE or any other kernel-level debugger (RR0D, BugChecker, Syser, etc) to debug drivers on the same machine. These tools look abandoned but they're still worth trying.

Another alternative is to use Linux host + qemu-kvm + PCI pass-through, this way suits me better.

**Comment by leecher1337** [comment]
I used a second PC with an ESS card via LAN connection to debug via Ethernet with WinDbg, faster than serial connection

**Comment by leecher1337** [comment]
For experimenting, I'd use my https://github.com/leecher1337/esfmbank/ bank editor. ;-)
